schemas/process.yml

```diff
@@ -197,3 +239,94 @@
+        For example, if process.group_leader.same_as_process = true, it means the process event in question is the leader of its process group. Details under process.* like pid would be the same under process.group_leader.*
+        The same applies for both process.session_leader and process.entry_leader.
+        This field exists to the benefit of EQL and other rule engines since it's not possible to compare equality between two fields in a single document.
+        e.g
+        process.entity_id = process.group_leader.entity_id (top level process is the process group leader)
+        OR
+        process.entity_id = process.entry_leader.entity_id (top level process is the entry session leader)
+        Instead these rules could be written like:
+        process.group_leader.same_as_process: true
+        OR
+        process.entry_leader.same_as_process: true
+        Note: This field is only set on process.entry_leader, process.session_leader and process.group_leader.
+
+    - name: env_vars
+      level: extended
+      type: object
+      beta: This field is beta and subject to change.
+      short: Environment variables set at the time of the event.
+      description: >
+        Environment variables set at the time of the event.
+        May be filtered to protect sensitive information.
+      example: >
+        {
+          "USER": "elastic",
+          "LANG": "en_US.UTF-8",
+          "HOME": "/home/elastic"
+        }
+
+    - name: entry_meta.type
+      level: extended
+      type: keyword
+      beta: This field is beta and subject to change.
+      short: The entry type for the entry session leader.
+      description: >
+        The entry type for the entry session leader.
+        Values include: init(e.g systemd), sshd, ssm, kubelet, teleport, terminal, console
+
+    - name: entry_meta.source
+      level: extended
+      type: source
+      beta: This field is beta and subject to change.
+      short: Entry point information for a session.
+      description: >
+        Entry point information for a session.
+        Remote client information such as ip, port and geo location.
+
+    - name: tty
+      level: extended
+      type: object
+      beta: This field is beta and subject to change.
+      short: Information about the controlling TTY device.
+      description: >
+        Information about the controlling TTY device. If set, the process belongs to an interactive session.
+        
+    - name: tty.char_device.major
+      level: extended
+      type: long
+      beta: This field is beta and subject to change.
+      short: The TTY character device's major number.
+      description: >
+        The major number identifies the driver associated with the device. The character device's major and minor numbers can be algorithmically combined to produce the more familiar terminal identifiers such as "ttyS0" and "pts/0. For more details see https://www.kernel.org/doc/html/v4.11/admin-guide/devices.html
```

Comment:
Nit: Can we drop the reference URL in favor of something generic: `For more details, refer to the Linux kernel documentation`?

Saves ourselves from having to remove a broken link if the URL changes in the future.